specification/chaos/resource-manager/Microsoft.Chaos/preview/2024-03-22-preview/experiments.json

```diff
@@ -0,0 +1,547 @@
+      "post": {
+        "description": "Cancel a running Experiment resource.",
+        "tags": [
+          "Experiments"
+        ],
+        "operationId": "Experiments_Cancel",
+        "parameters": [
+          {
+            "$ref": "./types/common.json#/parameters/apiVersion"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "./types/experiments.json#/parameters/experimentName"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Long running cancel operation."
+          },
+          "default": {
+            "description": "Error response returned if request was unsuccessful.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Cancel a running Experiment.": {
+            "$ref": "./examples/CancelExperiment.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Chaos/experiments/{experimentName}/start": {
+      "post": {
+        "description": "Start a Experiment resource.",
+        "tags": [
+          "Experiments"
+        ],
+        "operationId": "Experiments_Start",
+        "parameters": [
+          {
+            "$ref": "./types/common.json#/parameters/apiVersion"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "./types/experiments.json#/parameters/experimentName"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Long running start experiment operation."
+          },
+          "default": {
+            "description": "Error response returned if request was unsuccessful.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Start a Experiment.": {
+            "$ref": "./examples/StartExperiment.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Chaos/experiments/{experimentName}/executions": {
```

Comment:
>executions [](http://example.com/codeflow?start=127&length=10)

This name change doesn't look good to me. In terms of negative connotatiosn of 'execution'. Why take it?

Reply:
I assume this is approved as a bugfix?

Reply:
no this is not the bugfix. this is merging our stable branch into our preview branch. We've had this endpoint in our stable branch for a two versions now.

Reply:
As far as the negative connotations associated with execution. We have used this in stable for two versions now with SDK releases. Are you advocating to have us go through a breaking change to update it?